src/validator/AdminValidator.cpp

```diff
@@ -92,8 +92,12 @@ Status CreateSpaceValidator::validateImpl() {
                 } else {
                     spaceDesc_.set_isolation_level(meta::cpp2::IsolationLevel::DEFAULT);
                 }
+                break;
             }
             case SpaceOptItem::GROUP_NAME: {
+                if (item->getGroupName() == "default") {
```

Comment:
Use `folly::toLowerAscii` to deal with, and then compare ？

Reply:
Because group name case sensitive, so it's only limit the `default`

Reply:
`default `as the group name, should ignore the case sensitive， WDYT